docs/api-docs/tax/tax-rates-zones.md

```diff
@@ -16,7 +16,7 @@ You can further narrow the scope of a zone by specifying one or more customer gr
 
 <!-- theme: info -->
 > #### Default tax zone
-> A default tax zone covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone fall under the default tax zone. A default tax zone does not have `shopper_target_settings` in its requests or responses. You can't specify the customer groups or locations for a default tax zone.  
+> The default tax zone, which has a zone `id` of 1, covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone or shoppers that don't belong in a customer group fall under the default tax zone. You cannot disable or delete the default tax zone. When updating the default tax zone, you cannot adjust the locations or customer groups that it covers.  
```

Comment:
```suggestion
> The default tax zone, which has a zone `id` of 1, covers locations that other zones don't cover. Shoppers who don't meet the criteria for any manually-defined tax zone fall under the default tax zone. You cannot disable or delete the default tax zone. When updating the default tax zone, you cannot adjust the locations or customer groups that it covers.  
```
is there a reason that we needed to mention the customer groups at all here? I understand there's a bit of complexity re: tax zones generally and tax zones for groups, but I think that's covered elsewhere in the article.

Reply:
Good point, the catch-all statement is less complicated for users, will use it